### DISCUSSION TITLE
Swagger Error

### DISCUSSION THREAD

**Discussion by drakub** [YetiForceCompany/YetiForceCRM#16391] [General]
Hello, I have been trying to access the yetiforce API documentation on swagger but I keep getting the below error. What could be wrong?
![image](https://user-images.githubusercontent.com/19745222/186860454-70a4fd8b-b8df-4cc5-84a8-2cf648b0a219.png)

**Comment by rskrzypczak** [comment]
Recently, we made some changes in the context of api documentation availability.
We will restore the website availability soon.

**Reply by drakub** [reply]
Well, in the mean time, I wanted to look at how to download file(s) or document(s) related to a particular record say Ticket H229018 using an API, how do I do this??

**Reply by rskrzypczak** [reply]
You can always load swagger documentation in an independent online editor e.g. https://editor.swagger.io/
here you have the files: https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/api/doc

**Reply by drakub** [reply]
Thanks, this loaded

**Comment by mariuszkrzaczkowski** [comment]
https://doc.yetiforce.com/api/ it's working now
